bootstrap/090_post_setup.sql

```diff
@@ -379,6 +379,28 @@ CREATE OR REPLACE TASK TASKS.WAREHOUSE_SCHEDULING
     as
     call INTERNAL.UPDATE_WAREHOUSE_SCHEDULES(NULL, NULL);
 
+CREATE OR REPLACE TASK TASKS.SUNDECK_DEBUG
+    ALLOW_OVERLAPPING_EXECUTION = FALSE
+    USER_TASK_MANAGED_INITIAL_WAREHOUSE_SIZE = "X-Small"
+    AS
+BEGIN
+    let org_name text := (select current_organization_name());
+    IF (org_name <> 'SUNDECK') THEN
+        RETURN false;
+    END IF;
+
+    CREATE OR REPLACE PROCEDURE admin.run_as_app(sql string)
```

Comment:
With this get the right perms?

Reply:
hrm, probably need to add a grant after making it. I bet I have my local app in "DEBUG" mode which is why I didn't get an error. I'll re-test that and add a grant.

Reply:
I can make the procedure in the task, but the task user (SYSTEM) doesn't have the permission to directly run the grant. 

Maybe the task can call an internal procedure (which is already run as owner) and execute the grant?

moot point if I can make your other suggestion work.